backend/geonature/utils/config_schema.py

```diff
@@ -255,9 +255,78 @@ class GnFrontEndConf(Schema):
 
 
 class Synthese(Schema):
+    # --------------------------------------------------------------------
+    # SYNTHESE - SEARCH FORM
     AREA_FILTERS = fields.List(
         fields.Dict, load_default=[{"label": "Communes", "type_code": "COM"}]
     )
+    # Nombre de résultat à afficher pour la rechercher autocompleté de taxon
+    TAXON_RESULT_NUMBER = fields.Integer(load_default=20)
```

Comment:
Est-ce que ça serait pas plus parlant ?
```suggestion
    TAXON_AUTOCOMPLETE_ITEMS_COUNT = fields.Integer(load_default=20)
```

Reply:
> Est-ce que ça serait pas plus parlant ?

Ici aussi le paramètre `TAXON_RESULT_NUMBER` est un paramètre existant qui a juste été déplacé.